### PR TITLE
Fix invalid subscription status message in playlist item [stage-3]

### PR DESCRIPTION
**Pull request by olegrise**
Fix for #871 

@Rise-Vision/apps please review

### CODE REVIEW

File: test/unit/editor/directives/dtv-artboard-presentation.tests.js [hunk 1]
```diff
@@ -28,6 +28,7 @@ describe('directive: artboard-presentation', function() {
         presentation: presentation
       };
     });
+    $provide.service('playerLicenseFactory', function() {});
```
Comment (alex-deaconu) [inline test/unit/editor/directives/dtv-artboard-presentation.tests.js]: @olegrise why is this mock necessary here? Thanks.

File: test/unit/editor/controllers/ctr-workspace.tests.js [hunk 2]
```diff
@@ -11,6 +11,7 @@ describe('controller: Workspace', function() {
         }
       };
     });
+    $provide.service('playerLicenseFactory', function() {});
```
Comment (alex-deaconu) [inline test/unit/editor/controllers/ctr-workspace.tests.js]: @olegrise same question as below, re necessity of this mock. Thanks.

Comment (olegrise) [reply test/unit/editor/controllers/ctr-workspace.tests.js]: @alex-deaconu same - without it unit tests are failing.

File: test/unit/editor/directives/dtv-artboard-presentation.tests.js (hunk 1 above)
Comment (olegrise) [reply test/unit/editor/directives/dtv-artboard-presentation.tests.js]: @alex-deaconu I don't know exactly, but without it unit tests are failing.

Comment (alex-deaconu) [reply test/unit/editor/directives/dtv-artboard-presentation.tests.js]: @olegrise the `playerLicenseFactory` is part of the `'risevision.displays.services'` module. If you import that module to the editor it should resolve this conflict. Thanks.

**Comment by alex-deaconu** [comment]
@olegrise fix looks good to me. It works as intended. To confirm, the notification is gone from the Placeholder Playlist, and the Playlist Item Settings modal says "Subscribed".

Personally I think it would be more intuitive if it said "For Licensed Displays" so users understand why the Widget may or may not be working on a specific Display. @ezequielc thoughts? Thanks.

**Comment by ezequielc** [comment]
@olegrise Looks good. I agree with @alex-deaconu 's suggestion on changing the text to "For Licensed Displays". Thanks

**Comment by olegrise** [comment]
Thanks @ezequielc. @alex-deaconu what do you recommend - just change "Subscribed" to "For Licensed Displays" in [here](https://github.com/Rise-Vision/rise-vision-apps/pull/892/files#diff-a1f5f08d42c0cebfe79b03a7fc603facR379) or something else?

**Comment by olegrise** [comment]
@alex-deaconu @ezequielc please review last commit. Thanks.

**Comment by alex-deaconu** [comment]
Re-submitted https://github.com/Rise-Vision/rise-vision-apps/pull/894 for review. Closing. Thanks.

**Pull request closed without merging**